readme.md

```diff
@@ -72,19 +72,19 @@ If the latest version of the application wasn't yet pulled from [GitHub Releases
 
 Does the same as `/` (basically an alias).
 
-### /download/:platform
+### /download/:platform/:arch
 
-Accepts a platform (like "darwin" or "win32") to download the appropriate copy your app for. I generally suggest using either `process.platform` ([more](https://nodejs.org/api/process.html#process_process_platform)) or `os.platform()` ([more](https://nodejs.org/api/os.html#os_os_platform)) to retrieve this string.
+Accepts a platform (like "darwin" or "win32") and an architecture (like "ia32", or "x64") to download the appropriate copy your app for. I generally suggest using either `process.platform()` ([more](https://nodejs.org/api/process.html#process_process_platform)) and `process.arch()` ([more](https://nodejs.org/api/process.html#process_process_arch)) to retrieve these strings.
 
-If the cache isn't filled yet or doesn't contain a download link for the specified platform, it will respond like `/`.
+If :arch parameter is missing, `x64` is used by default.
```

Comment:
I think that a 404 is returned if cache isn't filled for this platform. See https://github.com/zeit/hazel/blob/master/lib/routes.js#L80